### PR TITLE
Feat:FE-60/translate the zurichat reset pasword page,Feat:FE-67/translate zurichat privacy policy page

### PR DESCRIPTION
**Pull request by Dannygbads**
https://linear.app/team-mallet/issue/FE-67/translate-zurichat-privacy-policy-page
https://linear.app/team-mallet/issue/FE-60/translate-the-zurichat-reset-pasword-page
FE-60
FE-67

**My PR Fixes FE-60/translate the zurichat reset pasword page
My PR Fixes FE-67/translate zurichat privacy policy page

# Changes proposed

## What were you told to do?
To Translate reset password page
to translate zurichat privacy policy page

## What did you do?
I added new key maps for other international languages which were:
arabic
chinese
deutch
dutch
hebrew
spanish
italian
french
portugese
I also added functionality to make hebrew and arabic texts spawn from right

# Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

Web capture_25-11-2022_20112_localhost
Web capture_25-11-2022_20951_localhost
Web capture_25-11-2022_201030_localhost
Web capture_25-11-2022_201112_localhost
Web capture_25-11-2022_201147_localhost
Web capture_25-11-2022_201214_localhost
Web capture_25-11-2022_201222_localhost
Web capture_27-11-2022_19551_localhost
Web capture_27-11-2022_195417_localhost
Web capture_27-11-2022_195430_localhost
Web capture_27-11-2022_195446_localhost

### CODE REVIEW

File: packages/main/src/pages/reset-password/ChangePassword.jsx [hunk 1]
```diff
@@ -75,9 +75,19 @@ const NewPassword = () => {
       updatepass();
     }
   };
+  const textAlign = () => {
```
Comment (ickynavigator) [inline packages/main/src/pages/reset-password/ChangePassword.jsx:78]: don't duplicate this function
it's the same as the one in the other file

**Review by ickynavigator — CHANGES_REQUESTED** [review]

File: packages/main/src/pages/reset-password/ChangePassword.jsx (hunk 1 above)
Comment (Dannygbads) [reply packages/main/src/pages/reset-password/ChangePassword.jsx]: i already removed it

Comment (Dannygbads) [reply packages/main/src/pages/reset-password/ChangePassword.jsx]: please review it again i already fixed the code

File: packages/main/src/pages/static/contact-us/faq/detailsArray.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 const Topics = [
   {
-    topic: "topic_adding_removing_users",
```
Comment (ickynavigator) [inline packages/main/src/pages/static/contact-us/faq/detailsArray.js:3]: isn't this necessary for the translation?

File: packages/root-config/src/applications.js [hunk 3]
```diff
@@ -1,8 +1,19 @@
 const PluginImportPathPre = "zuri-plugin-";
 const PluginAppPathPre = "plugin-";
 
+const textAlign = () => {
```
Comment (ickynavigator) [inline packages/root-config/src/applications.js:4]: isn't this function not already implemented?

Comment (ickynavigator) [inline packages/root-config/src/applications.js:4]: also why are you working in the root config???

File: packages/root-config/src/zuri-root-config.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ import { coreApps, allPlugins } from "./applications";
 
 coreApps.forEach(app => {
   registerApplication({
-    name: `@zuri/${app.name}`,
+    name: `@zuri${app.class}/${app.name}`,
```
Comment (ickynavigator) [inline packages/root-config/src/zuri-root-config.js:13]: whyyyy????

**Review by ickynavigator — CHANGES_REQUESTED** [review]
why are you in the root config????

File: packages/root-config/src/applications.js (hunk 3 above)
Comment (Dannygbads) [reply packages/root-config/src/applications.js:4]: It has been removed from the other file

Comment (Dannygbads) [reply packages/root-config/src/applications.js:4]: I put a functionality to apply text align right to Arabic and Hebrew texts

File: packages/root-config/src/zuri-root-config.js (hunk 4 above)
Comment (Dannygbads) [reply packages/root-config/src/zuri-root-config.js:13]: To apply styling for text align right for Arabic and Hebrew texts

Comment (Dannygbads) [reply packages/root-config/src/zuri-root-config.js]: please i already made the changes please review it again I'm sorry

File: packages/root-config/src/applications.js (hunk 3 above)
Comment (Dannygbads) [reply packages/root-config/src/applications.js]: i've already made the chages please review

File: packages/main/src/pages/static/contact-us/faq/detailsArray.js (hunk 2 above)
Comment (Dannygbads) [reply packages/main/src/pages/static/contact-us/faq/detailsArray.js:3]: it has been changed please review

File: packages/root-config/src/applications.js (hunk 3 above)
Comment (Dannygbads) [reply packages/root-config/src/applications.js]: i have removed the function completely please review

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**